UserAgent/Views/Privacy Indicator/PrivacyDashboardUtils.swift

```diff
@@ -1,125 +1,110 @@
 import Shared
 
 enum PrivacyDashboardUtils {
-    static func stackViewForStatEmpty(
-        _ color: UIColor,
-        _ text: String
-    ) -> UIStackView {
-        let dot = dotView(color)
-        let label = statLabel(text)
-
-        let stackView = UIStackView(arrangedSubviews: [dot, label])
-        stackView.spacing = 5
-        stackView.alignment = .top
-        return stackView
-    }
-
-    static func stackViewForStat(
-        _ color: UIColor,
-        _ text: String,
-        _ value: String
-    ) -> UIStackView {
-        let dot = dotView(color)
-        let label = statLabel(text)
-        let number = statLabel(value)
-        number.alpha = 0.7
-        number.textAlignment = .right
-
-        let stackView = UIStackView(arrangedSubviews: [dot, label, number])
-        stackView.spacing = 5
-        stackView.alignment = .top
-        stackView.distribution = .fillProportionally
-
-        label.snp.makeConstraints { make in
-            make.width.lessThanOrEqualTo(100)
+    class HStack: UIStackView {
+        required init(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
         }
-        number.snp.makeConstraints { make in
-            make.width.lessThanOrEqualTo(25)
-            make.width.greaterThanOrEqualTo(18)
+        init(_ callback: () -> (UIView, UILabel)) {
```

Comment:
Cannot this be done in more generic way? Imaging a callback return an array of UIViews.

Reply:
Array would be not restricted. Client can pass array with one element --> what should library do, throw exception? 
Now if client wants to pass four views it would typed error.

By the way, I dont think passing array it's a more generic way. Library still needs to write something like if (arr.count === 2) or if (arr.count === 3) I think overloading a bit better approach in this case